qiskit/circuit/library/standard_gates/rx.py

```diff
@@ -90,15 +92,23 @@ def control(
             ctrl_state: control state expressed as integer,
                 string (e.g.``'110'``), or ``None``. If ``None``, use all 1s.
             annotated: indicates whether the controlled gate can be implemented
```

Comment:
a bit out of scope, but does this indicate whether it can be implemented or whether it should be implemented? It looks to me like the second.

Reply:
I also think the second is correct... I can change it if we're already touching it 👍🏻

Reply:
Done in 7a03c4f